features/step_definitions/outcome.rb

```diff
@@ -84,3 +84,14 @@
   expect(highlighted_element).not_to be_nil
 end
 
+Alors('je peux lire uniquement {string} dans la liste des types') do |text|
+  expect(find('.list-group')).to have_content text
+  expect(find('.list-group')).not_to have_content "Ethnography/Report"
+  expect(find('.list-group')).not_to have_content "Ethnography/Analysis"
```

Comment:
Good examples: they could be in the scenario...

Reply:
Those information are now present in the scenario instead